server/bin/utils/run-unittest

```diff
@@ -580,8 +581,29 @@ function _run_test {
     _dump_logs
     _verify_output ${testname}
     _reset_state ${testname}
-    return 0
+
+    pass_or_fail=$(cat ${_testres})
+    if [[ ${pass_or_fail} == "PASS" ]]; then
+        echo "${pass_or_fail} - ${testname} ($(cat ${_testdur}))"
+        rm ${_testres}
+        rm -f ${_testdur}
+        rm -f ${_test_db}
+        rmdir ${_testroot}
+        res=0
+    else
+        if [[ -e ${_testdiff} ]]; then
+            cat ${_testdiff}
+        else
+            echo "+++"
+            echo "*** ${testname} failed with no output"
+            echo "---"
+        fi
+        echo "FAIL - ${testname} ($(cat ${_testdur}))"
+        res=1
+        rm -f ${_testdur}
```

Comment:
What about `_testres` and/or `_test_db`?

Reply:
Yes, so that one can go back and look at the data related to the failure.